torch/utils/hipify/hipify_python.py

```diff
@@ -130,16 +130,21 @@ def preprocess(
 
     # Preprocessing statistics.
     stats = {"unsupported_calls": [], "kernel_launches": []}
+    translated_files = {}
 
     for filepath in all_files:
         result = preprocessor(output_directory, filepath, stats, hip_clang_launch, is_pytorch_extension)
+        if result == "ok":
+            translated_files[filepath] = os.path.join(output_directory, get_hip_file_path(filepath))
+
         # Show what happened
         if show_progress:
             print(
                 filepath, "->",
                 get_hip_file_path(filepath), result)
 
     print(bcolors.OKGREEN + "Successfully preprocessed all matching files." + bcolors.ENDC, file=sys.stderr)
+    return translated_files
```

Comment:
This looks wrong

Reply:
If we decide to not remove the temp files, I can drop it again.
But then returning the files you created seems a relatively benign return value.

Reply:
I thought it looked wrong because you were unconditionally short-circuiting (with the return here), but there was dead code after this function.

Reply:
Right. Who needs debug output. :)
This is gone now along with the return value.